modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/StepSettings.scala

```diff
@@ -59,7 +59,8 @@ final case class ExposureTimeCell(s: Step, i: Instrument) extends ReactProps {
 object ExposureTimeCell {
   type Props = ExposureTimeCell
 
-  implicit val propsReuse: Reusability[Props] = Reusability.derive[Props]
+  implicit val propsReuse: Reusability[Props] =
+    Reusability.by(p => (p.s.config, p.i))
```

Comment:
Why not `derive`?

Reply:
`derive` will compare all the `Step` properties, and rerender if any of them changes. In this component we only need to rerender if the `config` changes (and even then, only if the exposure time changes).